src/platform/plugins/shared/discover/public/context_awareness/profile_providers/observability/observability_profile_providers.ts

```diff
@@ -17,5 +18,6 @@ export const createObservabilityDocumentProfileProviders = (
   return [
     createObservabilityLogDocumentProfileProvider(providerServices),
     createObservabilityTracesDocumentProfileProvider(providerServices),
+    createObservabilityGenericDocumentProfileProvider(providerServices),
```

Comment:
Do we really need a new profile for this? We can simply update the `createObservabilityTracesDocumentProfileProvider` 

something like: 
```
    if (
      isObservabilitySolutionView &&
      (isGenericDocument(record) ||
        isTraceDocument(record, tracesContextService.isTracesIndexPattern))
    ) {
```
and the rest will come for free